localstack/services/apigateway/apigateway_starter.py

```diff
@@ -13,6 +13,20 @@
 
 
 def apply_patches():
+    apigateway_models_Stage_init_orig = apigateway_models.Stage.__init__
+
+    def apigateway_models_Stage_init(
+        self, name=None, deployment_id=None, variables=None, description='',
+        cacheClusterEnabled=False, cacheClusterSize=None
+    ):
+        apigateway_models_Stage_init_orig(self, name=None, deployment_id=None, variables=None, description='',
+            cacheClusterEnabled=False, cacheClusterSize=None)
+
+        if cacheClusterSize or cacheClusterEnabled:
```

Comment:
nit: Perhaps better change this to (to avoid overwriting if status exists):
```
if (cacheClusterSize or cacheClusterEnabled) and not self.get('cacheClusterStatus'):
    ...
```